### ISSUE TITLE
Transform the DHCP Helper CLI tool into a service

### ISSUE THREAD

**Issue by MoritzThomasHuebner** [magma/magma#14970]
In #14418 and #14528 we refactored the DHCP IP allocation logic into a independent CLI tool. However in #14724 #14881, and #14887 we noticed performance issues. Each concurrently running CLI call takes about 50 MB of memory overhead, so we had to limit the number of concurrent release processes to no more than 3 to safely fit within the 300 MB memory limit allocated to mobilityd.

In order to improve performance, we could refactor the CLI tool into a new service which permanently runs and does not have the same memory overhead every time it adds an allocate or a release process. Before starting this change, we first need to analyze whether creating a service can be done efficiently and in a timely manner. The service will also require its own systemd and docker handling.

@nstng @maxhbr

**Comment by shahrahman-fb** [comment]
Closing this issue as part of a repository cleanup effort. This issue has had no activity for over 2 years.

If this issue is still relevant, please reopen it or file a new issue with updated information. We are actively maintaining the project again and want to focus on current, actionable issues.

See: #15833 (Phase 1 cleanup), #15828 (tracking issue)
